crypto/store/store_local.h

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
It should be locl.h or lcl.h - following the existing pattern - and that name should be in this PR - it makes little point adding in another example of not following the usual pattern and pushing it to another PR. Another PR to catch the less common instances and make them match makes sense - but this PR shouldn't dig the hole deeper.